app/v1/services/AmendAnnualSubmissionService.scala

```diff
@@ -31,16 +31,16 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class AmendAnnualSummaryService @Inject()(connector: AmendAnnualSummaryConnector) extends DesResponseMappingSupport with Logging {
+class AmendAnnualSubmissionService @Inject()(connector: AmendAnnualSubmissionConnector) extends DesResponseMappingSupport with Logging {
 
-  def amendAnnualSummary(request: AmendAnnualSubmissionRequest)(
+  def amendAnnualSubmission(request: AmendAnnualSubmissionRequest)(
     implicit hc: HeaderCarrier,
     ec: ExecutionContext,
     logContext: EndpointLogContext,
     correlationId: String): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
 
     val result = for {
-      desResponseWrapper <- EitherT(connector.amendAnnualSummary(request)).leftMap(mapDesErrors(desErrorMap))
+      desResponseWrapper <- EitherT(connector.amendAnnualSubmission(request)).leftMap(mapDesErrors(desErrorMap))
```

Comment:
I guess this code was already there just the naming's changed, but it could be simplified to:
`connector.amendAnnualSubmission(request).map(_.leftMap(mapDesErrors(desErrorMap)))`

(i.e. doesn't need the monad transformer stuff)